pkg/spec/createconfig.go

```diff
@@ -378,8 +378,12 @@ func (c *CreateConfig) GetContainerCreateOptions(runtime *libpod.Runtime) ([]lib
 			return nil, errors.Wrapf(err, "container %q not found", c.NetMode.ConnectedContainer())
 		}
 		options = append(options, libpod.WithNetNSFrom(connectedCtr))
-	} else if !rootless.IsRootless() && !c.NetMode.IsHost() && !c.NetMode.IsNone() {
-		postConfigureNetNS := (len(c.IDMappings.UIDMap) > 0 || len(c.IDMappings.GIDMap) > 0) && !c.UsernsMode.IsHost()
+	} else if !c.NetMode.IsHost() && !c.NetMode.IsNone() {
+		isRootless := rootless.IsRootless()
+		postConfigureNetNS := isRootless || (len(c.IDMappings.UIDMap) > 0 || len(c.IDMappings.GIDMap) > 0) && !c.UsernsMode.IsHost()
+		if isRootless && len(portBindings) > 0 {
+			return nil, errors.New("port bindings are not yet supported by rootless containers")
+		}
```

Comment:
No resolv.conf mount?

Reply:
the container has a writeable `resolv.conf`:

```console
$ bin/podman run --rm -ti alpine grep resolv.conf /proc/mounts
tmpfs /etc/resolv.conf tmpfs rw,seclabel,nosuid,nodev,relatime,size=1584636k,mode=700,uid=1000,gid=1000 0 0
```